### PR TITLE
Describe the W1019 warning

### PR DESCRIPTION
**Pull request by AlexJameson**
One cannot put snippets in the short description as it was done in W1016

image

### CODE REVIEW

File: api-reference/10 UI Components/Errors and Warnings/W1019.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ id: ErrorsUIWidgets.W1019
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs if the maximum filter query length exceeds the value specified in the [maxFilterQueryLength](/Documentation/21_1/ApiReference/UI_Components/dxTagBox/Configuration/#maxFilterQueryLength) property of the [TagBox](/Documentation/21_1/ApiReference/UI_Components/dxTagBox/).
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/Errors and Warnings/W1019.md]:
```suggestion
Occurs if the maximum filter query length exceeds the value specified in the [maxFilterQueryLength](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#maxFilterQueryLength) property of the [TagBox](/Documentation/ApiReference/UI_Components/dxTagBox/).
```

File: api-reference/10 UI Components/Errors and Warnings/W1019.md [hunk 2]
```diff
@@ -3,7 +3,9 @@ id: ErrorsUIWidgets.W1019
...
+
+Increase the **maxFilterQueryLength** value to meet your requirements.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/Errors and Warnings/W1019.md]: Move this sentence to the full description.

File: api-reference/10 UI Components/Errors and Warnings/W1020.md [hunk 3]
```diff
@@ -3,7 +3,9 @@ id: ErrorsUIWidgets.W1020
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs if the [shading](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/#shading) property is **false**. [hideEvent](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/hideEvent/) is ignored.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/Errors and Warnings/W1020.md]: It's unclear how the `hideEvent` and `shading` properties are connected and why they depend on each other.

File: api-reference/10 UI Components/Errors and Warnings/W1020.md [hunk 4]
```diff
@@ -3,7 +3,9 @@ id: ErrorsUIWidgets.W1020
...
+
+Assign **true** to the **shading** property.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/Errors and Warnings/W1020.md]: Move this sentence to the full description too.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: api-reference/10 UI Components/Errors and Warnings/W1019.md (hunk 1 above)
Comment (AlexJameson) [reply api-reference/10 UI Components/Errors and Warnings/W1019.md]: For now, the maxFilterQueryLength property exists only in 21.1. I'll fix the TagBox link.

File: api-reference/10 UI Components/Errors and Warnings/W1020.md [hunk 5]
```diff
@@ -3,7 +3,8 @@ id: ErrorsUIWidgets.W1020
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs if the [shading](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/#shading) property is **false**. In this case, the [hideEvent](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/hideEvent/) property's value is ignored.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/Errors and Warnings/W1020.md]: Does it occur any time `shading` is false?

Comment (AlexJameson) [reply api-reference/10 UI Components/Errors and Warnings/W1020.md]: No, I was wrong. It occurs any time when it is **true**

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**